hoodie-client/src/main/java/com/uber/hoodie/table/HoodieMergeOnReadTable.java

```diff
@@ -206,6 +206,29 @@ public Partitioner getUpsertPartitioner(WorkloadProfile profile) {
                   throw new UncheckedIOException("Failed to rollback for commit " + commit, io);
                 }
               case HoodieTimeline.DELTA_COMMIT_ACTION:
+                // --------------------------------------------------------------------------------------------------
+                // (A) The following cases are possible if index.canIndexLogFiles and/or index.isGlobal
+                // --------------------------------------------------------------------------------------------------
+                // (A.1) Failed first commit - Inserts were written to log files and HoodieWriteStat has no entries. In
```

Comment:
deleting logs is bit different from deleting parquet files.. Queries can pick the log file up and may crash in between if we delete them?

In case of parquet files, they are not published to the query, so deleting works.

Reply:
These are deleting log files for failed first commits. The queries should not pick up failed commits.